docs/code-examples/bar_chart.cpp

```diff
@@ -0,0 +1,17 @@
+// Log a batch of 3D arrows.
+
+#include <rerun.hpp>
+
+#include <filesystem>
+#include <iostream>
+#include <string>
+#include <vector>
+
+namespace rr = rerun;
```

Comment:
we should move away from the rr alias, just like we did in Python and Rust. Let's write it out as `rerun`